awx/ui_next/src/components/FormLayout/FormLayout.jsx

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+
```

Comment:
I like having all these *Layout components together in one file here. Makes it a lot easier to see how their styles work together. Adding a few suggested changes below: